label-application-rule-config-service-impl/src/main/java/org/hypertrace/label/application/rule/config/service/ConfigServiceCoordinator.java

```diff
@@ -0,0 +1,16 @@
+package org.hypertrace.label.application.rule.config.service;
+
+import java.util.List;
+import org.hypertrace.core.grpcutils.context.RequestContext;
+import org.hypertrace.label.application.rule.config.service.v1.LabelApplicationRule;
+
+public interface ConfigServiceCoordinator {
```

Comment:
This pattern has been abstracted in IdentifiedObjectStore, you should be able to use that close to directly.

Reply:
okay.